### ISSUE TITLE
Can't get code coverage with nyc to work

### ISSUE THREAD

**Issue by unional** [avajs/ava#1181]
ava: 0.17.0
nyc: 10.0.0
coveralls: latest

I follow the instruction: https://github.com/avajs/ava/blob/master/docs/recipes/code-coverage.md

But couldn't get it to work.
Coveralls saw the builds from Travis CI but saying there is no data.

Here are some example repos:
https://github.com/unional/ava-fixture
https://travis-ci.org/unional/ava-fixture
https://coveralls.io/github/unional/ava-fixture

https://github.com/unional/global-store
https://travis-ci.org/unional/global-store
https://coveralls.io/github/unional/global-store

I have also asked this question on SO: http://stackoverflow.com/questions/41528849/nyc-coveralls-integration-not-working/41543013?noredirect=1#comment70335420_41543013

**Comment by sindresorhus** [comment]
You have to ensure you build the TypeScript code with source maps so `nyc` is able to translate the offsets.

**Comment by unional** [comment]
Thanks. I got it working.

Seems like the way I use the `sourceRoot` is a hack.

I configured it so that the resulting build have the right source map root in the browser:
![image](https://cloud.githubusercontent.com/assets/3254987/21840659/83760328-d793-11e6-8f74-db231dc815e3.png)

```js
{
  "compilerOptions": {
    "inlineSources": true,
    "sourceMap": true,
    "sourceRoot": "/ava-fixture"
  }
}
```

This is the setting I tried to get the source maps play nice on the browser.

When I removed it, the coverage info starts showing up.

Need to figure out how to do the source map translation on `webpack`...
